scripts/create-release-branch-workflow.sh

```diff
@@ -0,0 +1,7 @@
+CircleCIToken=$1
```

Comment:
Do we need to add `#!/usr/bin/env bash` to the bash scripts ?

Reply:
VSCode is still able to launch it via a task because it's specified as a shell script, but I will go ahead and it incase we need to launch it manually. Good catch